src/main/java/marquez/api/CreateJobRequest.java

```diff
@@ -0,0 +1,21 @@
+package marquez.api;
+
+import java.util.List;
+import javax.ws.rs.QueryParam;
+import lombok.Data;
+
+@Data
+public final class CreateJobRequest {
+
+  @QueryParam("location")
```

Comment:
is `QueryParam` what we want here? these fields are probably in the PUT/POST body?

Reply:
You're right. I imagine we don't have test coverage here, otherwise this would have been caught. Adding a todo-item for myself to add coverage for this.